src/components/tabs/tabs.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { FC, ReactNode, useEffect } from "react"
+import { StyledTabsWrapper, StyledTabs } from "./styled"
+import { useSetActive, useBuildTabs, OnChange } from "./tabs-hooks"
+
+export interface TabsProps {
+  className?: string
+  onChange?: OnChange
+  selected?: number
+  children?: ReactNode
+  TabContent?: string | React.ComponentType<any>
+}
+
+export const Tabs: FC<TabsProps> = ({
+  className,
+  onChange,
+  selected,
+  children,
+  TabContent = "div",
```

Comment:
Is this prop actually needed? Content itself could be some react node, right?

Reply:
Actually it's kind of an important one 😄 
Each active tab is getting rendered inside this tag, this means we can use a layout component without having to duplicate the code for each one.

Example:
```
const TabContent = props => <div style={{padding: 50, backgroundColor: red}}>{props.children}</div>

...
<Tabs TabContent={TabContent}>
  <Tab label="hi">Hello</Tab>
  <Tab label="Hi again">Hello again</Tab>
  <Tab label="Bye">Goodbye</Tab>
  <Tab label="Goodbye">Fairwell</Tab>
</Tabs>
```

Reply:
```
<Tabs>
  <Tab label="hi">
    <StyledTabComponent />
</Tab>
  <Tab label="Hi again">
     <AnotherComponent />
  </Tab>
</Tabs>
```
Usually tabs are more than just text, and they will have own container anyway, so we can simply remove this layer of control from Tabs component. I feel like this additional responsibility can be easily taken to the userland.

Reply:
I can use `React.Fragment` as default. Let's see if that's ok

Reply:
Let's do it like that then.

Reply:
Already pushed it ;)
Let's merge it!